fix_simulation/client1.cfg

```diff
@@ -0,0 +1,20 @@
+[DEFAULT]
+ConnectionType=initiator
+ReconnectInterval=60
+FileStorePath=store/client1
```

Comment:
You can disable the store file, set a flag to reset seqnums on logon, and setup cancel on disconnect if you want to simplify start/stop

Reply:
update in https://github.com/true-markets/tools/pull/1/commits/bc5c4b51e0f30aec79ea9038edf577a018d0bc46 . used `uat1.truex.co` as it was able to resolve. My guess is uat1 resolves to uat box 1 machine.
Also had to use the port for the rest api as the endpoint wont resolve without the port. 
e.g `curl -H "Content-Type: application/json"      -H "X-TRUEX-AUTH-TIMESTAMP: $(date +%s)"      http://uat1.truex.co:9742/api/v1/instrument` works but `curl -H "Content-Type: application/json"      -H "X-TRUEX-AUTH-TIMESTAMP: $(date +%s)"      http://uat1.truex.co:9742/api/v1/instrument` yields 
```
curl: (7) Failed to connect to uat1.truex.co port 80 after 218 ms: Couldn't connect to server
```